db_stress_tool/db_stress_shared_state.h

```diff
@@ -47,6 +47,7 @@ DECLARE_int32(open_read_fault_one_in);
 DECLARE_int32(inject_error_severity);
 DECLARE_bool(disable_auto_compactions);
 DECLARE_bool(enable_compaction_filter);
+DECLARE_uint32(commit_bypass_memtable_one_in);
```

Comment:
Just for my own learning, why move this flag definition into this header?

Reply:
Oh probably some change needed during development, fixed.